double/draft-ietf-perc-double.md

```diff
@@ -253,23 +253,25 @@ To encrypt a packet, the endpoint encrypts the packet using the inner (end-to-en
 cryptographic key and then encrypts using the outer (hop-by-hop) cryptographic
 key.  The processes is as follows:
 
-* Form an RTP packet.  If there are any header extensions, they MUST
+1. Form an RTP packet.  If there are any header extensions, they MUST
   use [@!RFC5285].
 
-* Form a synthetic RTP packet with the following contents:
+2. If the packet carries repair data, skip to step 6.
```

Comment:
I would be more generic than this.  

* Add some text to the opening part of the section saying 
  * that the transform can operate in single or double mode, 
  * ... depending on information provided per-packet by the calling code. 
  * this is used to allow devices with only the HBH key to inject packets when authorized in signaling (e.g., for repair)
* Have the repair section say that you need to flip the stack into single mode when decrypting a repair packet